src/util.js

```diff
@@ -72,7 +72,7 @@ export function maybeQuote(value, dialectName) {
     return 'NULL'
   }
 
-  if (typeof value === 'number') return value
+  if (typeof value === 'number' || typeof value === 'bigint') return value
   if (value && typeof value.toSQL === 'function') return value.toSQL()
   if (
     value instanceof Buffer &&
```

Comment:
I know this is a pretty low level utility but I think it'd be good to have at least one test somewhere that exercises this codepath just so it doesn't break if someone goes to change it. Because this is used pretty pervasively whatever test is easiest to write works for me!